crates/apps_lib/src/config/genesis/templates.rs

```diff
@@ -290,6 +292,8 @@ pub struct ChainParams<T: TemplateValidation> {
     pub max_block_gas: u64,
     /// Gas limit of a masp transaction paying fees
     pub masp_fee_payment_gas_limit: u64,
+    /// The amount of NAM the MASP shielding fee costs
+    pub masp_nam_shielding_fee: String,
```

Comment:
Don't we need a map here to track the costs for the non-native tokens?

Reply:
Since, we're arent going to support non-native tokens at genesis, I'm resolving this comment